lighthouse-core/report/v2/report-renderer.js

```diff
@@ -19,19 +19,190 @@
 
 window.ReportRenderer = class ReportRenderer {
   /**
-   * @param {!Object} report
+   * @param {Document} document
    */
-  constructor(report) {
-    this._report = report;
+  constructor(document) {
+    this._document = document;
   }
 
   /**
-   * @param {!Element} element
+   * @param {ReportJSON} report
+   * @return {Element}
    */
-  render(element) {
-    const pre = document.createElement('pre');
-    pre.textContent = JSON.stringify(this._report, null, 2);
-    element.appendChild(pre);
+  renderReport(report) {
+    try {
+      var element = this._renderReport(report);
```

Comment:
might have to make a decision if we follow devtools style and conventions or lighthouse (or drift the rest of lighthouse to devtools?). use of `const`, `let` vs. `var`, object shorthand, etc